test/imperative.js

```diff
@@ -38,6 +38,28 @@ metatests.testSync('sequential, must be possible to call end', (test) => {
   test.end();
 });
 
+metatests.testSync('sequential tests must be ordered', (test) => {
```

Comment:
We agreed not to use round brackets in single-argument lambdas

Reply:
Oh, we should definitely change the eslint config in this repo.